client/app/dedicated/server/dns/dedicated-server-dns.routes.js

```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-dns.html';
```

Comment:
Looks like all those routes could be generated automatically ;)

Reply:
Hmm why not, but how to do it ?